commit a0bb5abd09e3d0f66c3dccf70bf74c8b2d7cc1c2
Author: Samuel Thibault <samuel.thibault@ens-lyon.org>
Date:   Thu Jan 24 19:27:00 2019 +0100

    hurd: Fix initial sigaltstack state
    
            * hurd/hurdsig.c (_hurd_thread_sigstate): Set SS_DISABLE in
            sigaltstack.ss_flags.

Index: glibc-2.28/hurd/hurdsig.c
===================================================================
--- glibc-2.28.orig/hurd/hurdsig.c
+++ glibc-2.28/hurd/hurdsig.c
@@ -94,6 +94,7 @@ _hurd_thread_sigstate (thread_t thread)
       __sigemptyset (&ss->blocked);
       __sigemptyset (&ss->pending);
       memset (&ss->sigaltstack, 0, sizeof (ss->sigaltstack));
+      ss->sigaltstack.ss_flags |= SS_DISABLE;
       ss->preemptors = NULL;
       ss->suspended = MACH_PORT_NULL;
       ss->intr_port = MACH_PORT_NULL;
